schema/rpc.fbs

```diff
@@ -338,20 +347,73 @@ table AutoBoneProcessRequest {
 
 table AutoBoneProcessStatusResponse {
     process_type: AutoBoneProcessType;
-    message: string;
+    message: string (deprecated);
+    /// The current count. This value is -1 if there is nothing to report.
     current: uint32;
+    /// The total count. This value is -1 if there is nothing to report.
     total: uint32;
+    /// The time remaining in seconds. This value is -1 if there is nothing to report.
+    eta: float32;
```

Comment:
It looks like this field was inserted in the middle of the table, breaking all subsequent fields. Please move it to the end